rostsd_gen/index.js

```diff
@@ -0,0 +1,506 @@
+/* eslint-disable max-depth */
+/* eslint-disable no-sync */
+/* eslint-disable camelcase */
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// create interfaces.d.ts containing from each typeclass definition
+/* Example output for std_msgs_msg_String
+declare module "rclnodejs" {
+  namespace std_msgs {
+	  namespace msg {
+			export type String = {
+				data: string
+			}
+		}
+	}
+}
+*/
+
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+const rclnodejs = require('../index.js');
```

Comment:
I suspect this line causes the runtime error when loading the dynamic library. I haven't investigated why this only happens on macOS, probably we executed:

``` npm install --python=python2.7```

Reply:
You were correct. I removed line #34 which resolved this problem. Thx!